worlds/tracker/docs/map-integration.md

```diff
@@ -0,0 +1,92 @@
+# Map integration
+
+This document describes the requirements to add a map tab to UT. UT uses the [Poptracker](https://github.com/black-sliver/PopTracker/tree/master) specificiations for the content of the map tab.
+
+The intent of UT's map tab is to allow world devs to start making a poptracker pack before logic settles into a more stable configuration, however there's enough benifits to UT's integrated tab that it might be worth adding support even if the world already has a dedicated poptracker pack.
```

Comment:
```suggestion
The intent of UT's map tab is to allow world devs to start making a poptracker pack before logic settles into a more stable configuration, however there's enough benefits to UT's integrated tab that it might be worth adding support even if the world already has a dedicated poptracker pack.
```